docker-mender-convert

```diff
@@ -26,7 +26,7 @@ MENDER_CONVERT_DIR="$(pwd)"
 MENDER_CONVERT_VERSION=$(git_mender_convert_version)
 
 # Create a unique work directory so we can run multiple containers in parallel
-WORK_DIR=$(mktemp -d -p $(pwd) -t work.XXXX)
+WORK_DIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
```

Comment:
I think the directory has to be created as a subfolder of this directory, otherwise it will not be visible inside Docker. I have a couple of suggested workarounds. Either:

```
WORK_DIR=$(mktemp -d $(pwd)/work.XXXX)
```

or

```
WORK_DIR=$(env TMPDIR=$(pwd) mktemp -d work.XXXX)
```